tests/hotplug/memory.go

```diff
@@ -266,7 +271,7 @@ var _ = Describe("[sig-compute][Serial]Memory Hotplug", decorators.SigCompute, d
 			Expect(reqMemory).To(BeNumerically(">=", guest.Value()))
 
 			By("Hotplug additional memory")
-			newMemory := resource.MustParse("256Mi")
+			newMemory := resource.MustParse("1042Mi")
```

Comment:
guess you meant 1024?

Reply:
1024Mi wouldn't be an hotplug, I actually meant 1042Mi.